### PR TITLE
updated point symbols doc to reflect current bulk wkt behaviour

### PR DESCRIPTION
**Pull request by mtCarto**
@jodygarnett Can you give this a quick review?

### CODE REVIEW

File: doc/en/user/source/styling/sld-extensions/pointsymbols.rst [hunk 1]
```diff
@@ -304,6 +288,35 @@ For example:
...
+
+.. code-block:: xml
+   :linenos: 
+
+    <PointSymbolizer>
+      <Graphic>
+        <ExternalGraphic>
+          <OnlineResource 
+            xlink:type="simple"
+            xlink:href="example.properties" />
```
Comment (jodygarnett) [inline doc/en/user/source/styling/sld-extensions/pointsymbols.rst]: where is {{zig}} or {{block}} in this example!

**Comment by mtCarto** [comment]
Whoops! Added some Zig to the example

**Comment by jodygarnett** [comment]
I will backport to 2.9.x as there are no code changes

File: doc/en/user/source/styling/sld-extensions/pointsymbols.rst [hunk 2]
```diff
@@ -304,6 +288,35 @@ For example:
...
+   block=POLYGON((0 0, 1 0, 1 1, 0 1, 0 0))
+
+The SLD to use the symbols defined in :file:`example.properties` is:
+
+.. code-block:: xml
+   :linenos: 
+
+    <PointSymbolizer>
+      <Graphic>
+        <ExternalGraphic>
```
Comment (aaime) [inline doc/en/user/source/styling/sld-extensions/pointsymbols.rst]: This looks definitely wrong, WKTMarkFactory emits a shape, how is one going to assign stroking and filling to it? I'm for one surprised this works at all, but if it does, the code needs fixing, not the documentation. The previous documentation correctly documents the intended behavior and usage, there might be issues with relative paths not being recognized.

**Comment by aaime** [comment]
@jodygarnett please do not backport a faulty example to 2.9.x, fix the code instead.

**Pull request merged**